Source/Fuse.Android/SystemScreenConfig.uno

```diff
@@ -0,0 +1,170 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse.Platform;
+
+namespace Fuse.Android
+{
+	/**
+		Allows you to control the state of the status bar and navigation bar on Android systems
+
+		# Note
+
+		 * You are not allowed to have the navigation bar visible alone. This is a general rule for Android, and is therefore enforced here
+		 * The user can change this state themselves, for example through swiping down from the top of the display. Fuse will automatically revert such changes after a time set by `ResetDelay`
+		 * Setting `ResetDelay` to 0 disables this behavior
+
+		# Example
+
+		The following example allows the user to control the status bar and navigation bar state through three buttons. If the user manually changes the visibility state of the bars, it will be reverted after 2 seconds.
+
+			<App>
+				<ClientPanel Color="#2196F3">
+					<Android.SystemScreenConfig Show="{displayValue}" ResetDelay="2"/>
+					<JavaScript>
+						var Observable = require("FuseJS/Observable");
+						var displayValue = Observable("All");
+						function status() {
+							displayValue.value = "Status";
+						}
+						function all() {
+							displayValue.value = "All";
+						}
+						function none() {
+							displayValue.value = "None";
+						}
+						module.exports = {
+							status: status,
+							all: all,
+							none: none,
+							displayValue: displayValue
+						}
+					</JavaScript>
+					<StackPanel>
+						<Text FontSize="20">
+							Status bar and navigation bar example
+						</Text>
+						<Button Clicked="{status}" Text="Status" Margin="20"/>
+						<Button Clicked="{all}" Text="All" Margin="20"/>
+						<Button Clicked="{none}" Text="None" Margin="20"/>
+					</StackPanel>
+				</ClientPanel>
+			</App>
+	*/
+	public class SystemScreenConfig : Behavior
```

Comment:
You should add `OnRooted` and `OnUnrooted` handlers. The actual changes should happen unless the node is rooted. The OnUnrooted must cleanup the Timer as well if it's still sticking around.